### PR TITLE
Update to use go modules, new k8s versions

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,24 +22,7 @@ DOCKER_RUN := mkdir -p .go-pkg-cache && \
 				 -v $${PWD}/.go-pkg-cache:/go/pkg:rw \
 				 -v $${PWD}/hack/boilerplate:/go/src/k8s.io/kubernetes/hack/boilerplate:rw \
 				 -w /go/src/github.com/tigera/api \
-				 -e GOARCH=$(ARCH)
```
Comment (caseydavenport) [inline Makefile:25]: Whoops, this was accidental

Comment (caseydavenport) [reply Makefile:25]: Nevermind... just github rendering the diff horribly.

**Comment by bcreane** [comment]
I don't see `go.mod` in the PR.

**Review by bcreane — APPROVED** [review]
looks good.

**Pull request merged**